packages/backend/src/app.ts

```diff
@@ -224,21 +228,31 @@ export class App {
       resolvers
     })
 
-    // Setup Apollo on graphql endpoint
+    // Setup Apollo
     this.apolloServer = new ApolloServer({
       schema: schemaWithResolvers,
-      context: async (): Promise<ApolloContext> => {
-        return {
-          container: this.container,
-          logger: await this.container.use('logger')
-        }
-      }
+      plugins: [ApolloServerPluginDrainHttpServer({ httpServer })]
     })
 
     await this.apolloServer.start()
-    koa.use(this.apolloServer.getMiddleware())
 
-    this.adminServer = koa.listen(port)
+    koa.use(bodyParser())
+
+    router.post(
+      '/graphql',
+      koaMiddleware(this.apolloServer, {
+        context: async (): Promise<ApolloContext> => {
+          return {
+            container: this.container,
+            logger: await this.container.use('logger')
+          }
+        }
+      })
+    )
```

Comment:
We need to specify a route for GraphQL, otherwise any endpoint (eg: `localhost:3001/foo/bar`) will be treated as a GraphQL request.

Without this route, this was failing: 
https://github.com/interledger/rafiki/blob/f8d41ddbc970272e94fc4794c907ef16ef7ddf47/packages/backend/src/graphql/resolvers/index.test.ts#L21

Reply:
At the moment the GraphQL playground is not working since we are only expecting the `POST` method for this endpoint. 

Should we do `router.all` to match any method or remove the router? Removing the router means that any endpoint will match the GraphQL request and we can remove the test mentioned above.

Reply:
I wouldn't expect GraphQL Playground to be working. It sounds like we'd need to instead switch to Apollo Sandbox.
Deciding what method(s) to have here would depend on what Apollo Sandbox needs.

https://www.apollographql.com/docs/apollo-server/api/plugin/landing-pages/#graphql-playground-landing-page

Reply:
> I wouldn't expect GraphQL Playground to be working. It sounds like we'd need to instead switch to Apollo Sandbox.

Sorry for the confusion. The "playground" in the newer versions of Apollo is an embedded version of Sandbox.

> It seems as though Apollo Sandbox should be working out of the box with the current setup as it stands... nothing jumps to mind based on the docs 🤔

The Sandbox is not working at the moment because I did this:
```ts
koa.use(router.routes()).use(router.allowedMethods())
```
and it was only expecting `POST` requests for that route.